### ISSUE TITLE
Inconsistent first line style between `simplenote-electron` and `simplenote-android`

### ISSUE THREAD

**Issue by wenfangdu** [Automattic/simplenote-android#1669]
`simplenote-electron`
<img width="680" alt="image" src="https://github.com/user-attachments/assets/388dddcb-f848-42c5-a87e-a6029370707c">
`simplenote-android`

![image](https://github.com/user-attachments/assets/87a95e96-39e5-469a-8825-d676759f3c56)

Compared to `simplenote-electron`, `simplenote-android`'s first line is bigger and bold, is that intentional?

Personally, I like the `simplenote-electron`'s approach, since it's simpler which is the main goal of Simplenote.

Any chance to align `simplenote-android`'s first line style with `simplenote-electron`'s?

**Comment by dangermattic** [comment]
Thanks for reporting! 👍

**Comment by roundhill** [comment]
> simplenote-android's first line is bigger and bold, is that intentional?

Yes, it's intentional. I think it's either a [bug in electron](https://github.com/Automattic/simplenote-electron/issues/3158) or it isn't possible to do it in the editor of the electron app. All of the other apps have the larger title for the first line. It's been this way for a long time, I don't think it's worth adding a setting for but maybe we could do something like if you type a backspace on the first line it removes the header styling and/or if the first line is above a certain character length.

**Comment by wenfangdu** [comment]
@roundhill I think it's because `simplenote-electron` is using `monaco-editor`, in this case, it's weird to have heading styles in a plain text editor.
